frontend/components/bc/BcMaintenanceBanner.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { warn } from 'vue'
+
+const { public: { maintenanceTS } } = useRuntimeConfig()
+const { tick } = useInterval(60)
+const { t: $t } = useI18n()
+
+const maintenanceLabel = computed(() => {
+  if (!maintenanceTS) {
+    return
+  }
+  const parsed = typeof maintenanceTS === 'number' ? maintenanceTS : parseInt(maintenanceTS)
+  if (isNaN(parsed)) {
```

Comment:
`parseInt()` is more tolerant than `Int()`. If the string contains "129,394,653" it returns 129 and ignores the rest... I believe.
`Int()` would return `Nan`.

Maybe, the `? :` is not necessary, I imagine that this line would work the same way:
```TS
const parsed = Int(maintenanceTS)
```

Reply:
Did not think about that, but I think it would be ok that... so at least we show the banner if someone added it with `,`